prac_09/band.py

```diff
@@ -0,0 +1,25 @@
+"""Band class for CP1404"""
+
+
+class Band:
+
+    def __init__(self, band_name):
+        self.band_name = band_name
+        self.musicians = []
+
+    def add(self, musician):
+        self.musicians.append(musician)
+        print(self.musicians)
+
+    def play(self):
+        for i in self.musicians:
```

Comment:
This is the epitome of poor naming. See: https://github.com/CP1404/Starter/wiki/Programming-Patterns#for-loops-definite-iteration